litmus-portal/frontend/src/pages/GetStartedPage/index.tsx

```diff
@@ -53,6 +54,31 @@ const GetStarted: React.FC = () => {
 
   const [loading, setIsLoading] = useState<boolean>(false);
 
+  // Checking if token is valid or not by finding the uid in database
+  const ValidateUser = () => {
+    fetch(`${config.auth.url}/getUser/${getUserId()}`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${getToken()}`,
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        if ('error' in data) {
+          console.error('User not found');
+          logout();
+        }
+      })
+      .catch((err) => {
+        console.error(err);
+        logout();
+      });
+    return true;
+  };
+
+  ValidateUser();
```

Comment:
Do we need this function call here? And if needed here, then it shouldn't be needed in onClick() of buttons?

Reply:
Yes, as soon as someone is redirected to /getStarted route, we check if the token details are valid. Also, whenever user enters a value in input field, the state of page changes and this function gets called. Therefore even if someone is already present on this page and tries to reset the password, then this will not allow them to do the changes. 
And since this function gets called on each input in the input field, therefore it is not required on onCompleted.